src/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@
 # for OS 10.9.
 #
 set( SCIFIOJars
-  https://downloads.openmicroscopy.org/bio-formats/5.0.4/artifacts/bioformats_package.jar
+  https://downloads.openmicroscopy.org/bio-formats/5.8.2/artifacts/bioformats_package.jar
   https://maven.imagej.net/service/local/repositories/releases/content/io/scif/scifio-itk-bridge/1.2.0/scifio-itk-bridge-1.2.0.jar
```

Comment:
I think it should be enough to: A) change the Bio-Formats version here to `5.9.2`, and B) change the scifio-itk-bridge version here to `1.2.1`.

Please test and let me know how it goes! If it works, amend this commit, and I'll merge it.